README copy.md

```diff
@@ -0,0 +1,30 @@
+# React + TypeScript + Vite
```

Comment:
I suggest that you remove the standard text from the README file - and you probably don't need this COPY file at all so it can be removed